include/swift/Parse/LocalContext.h

```diff
@@ -19,6 +19,7 @@
 #define SWIFT_PARSE_LOCALCONTEXT_H
 
 #include "llvm/ADT/DenseMap.h"
+#include "swift/AST/Identifier.h"
```

Comment:
What is this change for?

Reply:
This file is not included by "CompletionInstance.cpp" via "PersistentParserState.h" too. Since this file uses `Identifier`, it causes a compiler error without this.